dask/dataframe/groupby.py

```diff
@@ -827,10 +868,11 @@ def __init__(self, df, index, slice=None, **kwargs):
                 if isinstance(index, list):
                     if len(index) == 0:
                         raise ValueError("No group keys passed!")
-                    msg = "Grouper for '{0}' not 1-dimensional"
-                    raise ValueError(msg.format(index[0]))
+                    #msg = "Grouper for '{0}' not 1-dimensional"
+                    #raise ValueError(msg.format(index[0]))
```

Comment:
I adapted the logic to cover all currently supported index values and added a test case to make sure exceptions are raised for unsupported ones.